Product Backlog.md

```diff
@@ -12,18 +12,23 @@ This is a problem that many of us on our team have experienced, especially since
 ### Functional
 1. As a user, I want to create an account
 2. As a user, I want to log into my account
-3. As a user, I want to input and remove grocery items to my "kitchen/pantry"
+3. As a user, I want to be able to recover my password if I forget it or it becomes compromised
+4. As a user, I want to input grocery items to my "kitchen/pantry" by scanning barcodes
+5. As a user, I want to input grocery items to my "kitchen/pantry" by typing item names
+6. As a user, I want to be able to remove grocery items
+7. As a user, I want to keep track of my current supply of grocery items
+8. As a user, I want to find recipes based on items in my kitchen
+9. As a user, I want to add allergies to limit the recipes I look for
 4. As a user, I want to know the nutritional information about a recipe **(if time allows)**
 5. As a user, I want to be able to save the recipes that I like
 6. As a user, I want to read the recipe easily
 7. As a user, I want to rate recipes, and view the public's ratings of recipes **(if time allows)**
-8. As a user, I want to find recipes based on items in my kitchen
 9. As a user, I want to be able to sort recipes by culinary style and genre **(if time allows)**
 10. As a user, I want to be able to find recipes based on prep time **(if time allows)**
-11. As a user, I want to be able to recover my password if I forget it or it becomes compromised
-12. As a user, I want to receive suggestions for recipes based on whats in my pantry and what I have liked in the past
```

Comment:
I moved line 11 to the front along with signup and login all those first time stuff.
And line 12 we already have "receive recipes based on what the user has". And user preferences I think it's more of a developer thing, so I added "as a developer". I can change it back if it makes more sense